canister/src/state.rs

```diff
@@ -83,14 +83,20 @@ impl State {
         let unstable_blocks =
             UnstableBlocks::new(&utxos, stability_threshold, genesis_block, network);
 
+        let fees = match network {
+            Network::Mainnet => Fees::mainnet(),
+            Network::Testnet => Fees::testnet(),
+            _ => Fees::default(),
```

Comment:
nit: here and below I'd match over all the networks.

Reply:
Done. Thank you!